diff --git a/node_modules/react-native-screens/src/fabric/ScreenStackHeaderSubviewNativeComponent.ts b/node_modules/react-native-screens/src/fabric/ScreenStackHeaderSubviewNativeComponent.ts
index 61da7f7..1d09a80 100644
--- a/node_modules/react-native-screens/src/fabric/ScreenStackHeaderSubviewNativeComponent.ts
+++ b/node_modules/react-native-screens/src/fabric/ScreenStackHeaderSubviewNativeComponent.ts
@@ -12,7 +12,7 @@ export type HeaderSubviewTypes =
   | 'searchBar';
 
 export interface NativeProps extends ViewProps {
-  type?: CT.WithDefault<HeaderSubviewTypes, 'left'>;
+  type: CT.WithDefault<HeaderSubviewTypes, 'left'>;
   hidesSharedBackground?: boolean;
   synchronousShadowStateUpdatesEnabled?: CT.WithDefault<boolean, false>;
 }
